From 02966e348e37ebf6269aaed238e7ce67fbe958e7 Mon Sep 17 00:00:00 2001
From: Hrvoje Senjan <hrvoje.senjan@gmail.com>
Date: Sun, 25 May 2014 00:36:08 +0200
Subject: [PATCH 1/1] Drop Nepomuk from KParts' LINK_INTERFACE_LIBRARIES

Nepomuk is only used in a private header, browserrun_p.h,
thus it is not needed as KParts public dependancy
Makes it possible to drop libsoprano-devel from libkde4-devel Requires
---
 kparts/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kparts/CMakeLists.txt b/kparts/CMakeLists.txt
index 2eab2e8..e17ef5e 100644
--- a/kparts/CMakeLists.txt
+++ b/kparts/CMakeLists.txt
@@ -39,7 +39,6 @@ target_link_libraries(kparts  ${KDE4_KDECORE_LIBS} kdeui kio)
 target_link_libraries(kparts  LINK_PUBLIC kio kdeui kdecore ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} )
 if(HAVE_NEPOMUK)
   target_link_libraries(kparts  LINK_PRIVATE nepomuk nepomukutils)
-  target_link_libraries(kparts  LINK_PUBLIC nepomuk nepomukutils )
 endif(HAVE_NEPOMUK)
 
 set_target_properties(kparts PROPERTIES VERSION ${GENERIC_LIB_VERSION} 
-- 
1.9.3

